net-p2p/deluge/deluge-1.3.15-r3.ebuild

```diff
@@ -0,0 +1,134 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+PYTHON_COMPAT=( python2_7 )
+DISTUTILS_SINGLE_IMPL=1
+inherit distutils-r1 eutils systemd user
```

Comment:
and do nbot want `eutils` eclass

Reply:
Sorry, I've inherited it from old versions of the ebuild and I never noticed it wasn't used.

Done